### DISCUSSION TITLE
Hybrid App not allowing authorizations for other regions

### DISCUSSION THREAD

**Discussion by bpyzikvc** [amzn/selling-partner-api-models#3073] [General]
I have a developer and app listing that supports all marketplaces in all regions. However, when a user attempts to authenticate through the app marketplace, they receive the following error: 
"Something went wrong. The application provided does not have an MWS developer ID for this region. Error Code: MD9010". 

Do we need to switch our hybrid application to full SP-API? and if so, would this affect/require reauthorization of any of our current users? (We do not have any MWS users on it and have no need for that)

**Comment by sungolivia** [comment]
hmm you shouldn't need to, and you confirm that you associated that region's MWS dev ID to the hybrid app when you were migrating?
https://developer-docs.amazon.com/sp-api-blog/docs/troubleshooting-selling-partner-api-authorization-errors#error-codes
![Screen Shot 2022-10-04 at 1 00 00 PM](https://user-images.githubusercontent.com/7606170/193914139-f4523504-86c7-4e55-91eb-8494bb8a6e3a.png)

**Comment by bpyzikvc** [comment]
@sungolivia No we do not have another dev id to provide in that box. We ended up not migrating and started from scratch. If we made it an SP-API app only, those fields are removed. Would that fix the issue?

**Comment by bpyzikvc** [comment]
@sungolivia Any update on if that would fix our issue? and if that would cause any issues for existing users?
